css/1-layouts/footer.css

```diff
@@ -6,4 +6,63 @@
 
 .container .row .fourth {
     width: 25%
-}
\ No newline at end of file
+}
+
+.footer-logo { display: inline-block;  }
+
+ .fa {text-decoration: none;
+    color: black;
+    background-color: white;
+    padding: 10px 18px;
+    border-radius: 50%;}
+
+ .fa:hover {
+        background-color: blue;
+        color: white;
+    transition: 0.3s;}
+
+ 
+
+ul {list-style-type: none;}
+
+input {border: none;
```

Comment:
Man įtarimas dėl input. Ar jis nepaveiks kituose forum input sekcijose? Pasiulymas būtu patikslinti input selektoriu pvz. "footer-background input".

Reply:
It is commonly used to resetting defaults.  In case of custom view, u may  ned to create some more specific rule for it.
For ex. ul.custom {list-style-type:'-' ;

Reply:
Pofiksinau inputus dabar, turi but geriau